pre_build

```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
-
+#SOURCE_BRANCH=^release/.*$
+SOURCE_BRANCH=^test/.*$
+repo="usdotfhwastol/"
+export SOURCE_BRANCH
 if [[ "$SOURCE_BRANCH" = "develop" ]]; then
     # add -d flag to checkout.bash and update image dependencies
     sed -i "s|checkout.bash|checkout.bash -d|g; s|usdotfhwastol|usdotfhwastoldev|g; s|:[0-9]*\.[0-9]*\.[0-9]*|:develop|g" \
         Dockerfile
-elif [[ "$SOURCE_BRANCH" =~ ^release/.*$ ]]; then
+elif [[ "$SOURCE_BRANCH" =~ "^test/.*$" ]]; then
     # swap checkout branch in checkout.bash to release branch
-    sed -i "s|usdotfhwastol|usdotfhwastolcandidate|g; s|:[0-9]*\.[0-9]*\.[0-9]*|:candidate|g; s|CARMA[a-zA-Z]*_[0-9]*\.[0-9]*\.[0-9]*|candidate|g; s|carma-[a-zA-Z]*-[0-9]*\.[0-9]*\.[0-9]*|candidate|g" \
+    sed -i "s|$repo|usdotfhwastolcandidate/|g; s|:[0-9]*\.[0-9]*\.[0-9]*|:candidate|g; s|CARMA[a-zA-Z]*_[0-9]*\.[0-9]*\.[0-9]*|candidate|g; s|carma-[a-zA-Z]*-[0-9]*\.[0-9]*\.[0-9]*|candidate|g" \
```

Comment:
You are matching against usdotfhwastol/ now, but won't that fail to work with usdotfhwastoldev/ ?
You aught to be matching against all possible organizations since any of them could be specified in the docker file.
usdotfhwastol, usdotfhwastoldev, usdotfhwastolcandidate